.gitmodules

```diff
@@ -1,23 +1,23 @@
 [submodule "MCServer/Plugins/Core"]
-	path = MCServer/Plugins/Core
+	path = Server/Plugins/Core
 	url = https://github.com/mc-server/Core.git
 [submodule "MCServer/Plugins/ProtectionAreas"]
```

Comment:
Note: the submodule headers will have to stay like this (`MCServer/...`) in order to work with already-initialized repos.